performance/aws.py

```diff
@@ -8,3 +9,12 @@ def get_aws_session():
 def get_s3_resource():
     session = get_aws_session()
     return session.resource('s3')
+
+
+def get_s3_bucket():
+    return get_s3_resource().Bucket(config.S3_REPORTS_BUCKET)
```

Comment:
Can we make this function agnostic of the bucket (or, at least, take the `config.S3_REPORTS_BUCKET` as default but allow to pass a bucket name)? That would allow us to refactor `upload.py` to use this function, and to easily switch back to pushing to S3 instead of GSheets if needed.